docs/getting_started.md

```diff
@@ -816,18 +817,17 @@ To authenticate and decrypt a message:
 
 Mbed Crypto provides a simple way to generate a key or key pair.
 
-Prerequisites to using key generation and export APIs:
-* Initialize the library with a successful call to `psa_crypto_init`.
+**Prerequisites to using key generation and export APIs:**
+* Initialize the library with a successful call to `psa_crypto_init()`.
 
-Generate an ECDSA key:
+**To generate an ECDSA key:**
 1. Set the desired key attributes for key generation by calling
    `psa_set_key_algorithm()` with the chosen ECDSA algorithm (such as
-   `PSA_ALG_DETERMINISTIC_ECDSA(PSA_ALG_SHA_256)`). We don't set
-   `PSA_KEY_USAGE_EXPORT` as we only want to export the public key, not the key
+   `PSA_ALG_DETERMINISTIC_ECDSA(PSA_ALG_SHA_256)`). Do not set
+   `PSA_KEY_USAGE_EXPORT` because we only want to export the public key, not the key
```

Comment:
Maybe replace "because" with a ";". This is too easy to read as "Don't set it because of some reason (you may set it for some other reason)" instead of "Don't set it. Here's why you shouldn't set it."